### DISCUSSION TITLE
Deno as an optional backend

### DISCUSSION THREAD

**Discussion by trymeouteh** [tauri-apps/tauri#11893] [Ideas]
Would be cool to have Deno as a backend option.

- Deno is written in Rust!
- Deno is good on security with its permissions
- Deno is backward compatible with NodeJS

This way you can have the front end and back end written in JavaScript and not have to learn Rust.

**Comment by FabianLars** [comment]
This "feature" is tracked here https://github.com/tauri-apps/tauri/issues/4670 - deno would very likely be one of the first targets.
